### ISSUE TITLE
Confidence score for NER

### ISSUE THREAD

**Issue by ksindi** [explosion/spaCy#87]
Would it be possible to surface some relative score that a given tag is associated with its entity?

```
import spacy.en
nlp = spacy.en.English()
tokens = nlp(unicode(s, "utf-8"))
scores = [ent.score_ for ent in tokens.ents]  # confidence score
```

**Comment by ines** [comment]
There's now another thread with more info on this in #831 – closing this to keep the discussion in one place. This is a feature that will likely be implemented in spaCy v2.0.

**Comment by honnibal** [comment]
#881
